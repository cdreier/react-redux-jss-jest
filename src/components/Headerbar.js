import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

const Wrapper = styled.div`
  background-color: #333;
  padding: 10px;
  color: white;
`

const Headerbar = ({ children }) => {
  return (
    <Wrapper>
      {children}
    </Wrapper>
  )
}

Headerbar.propTypes = {
  children: PropTypes.any,
}


export default Headerbar
