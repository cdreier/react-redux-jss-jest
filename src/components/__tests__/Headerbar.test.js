import React from 'react'
import renderer from 'react-test-renderer'

import Headerbar from '../Headerbar'

it('renders the headerbar correctly', () => {
  const component = renderer.create(
    <Headerbar >Awesome title!</Headerbar>
  )
  expect(component.toJSON()).toMatchSnapshot()
})
