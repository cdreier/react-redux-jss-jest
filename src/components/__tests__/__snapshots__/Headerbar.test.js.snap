// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the headerbar correctly 1`] = `
<div
  className="sc-bdVaJa dcywn"
>
  Awesome title!
</div>
`;
